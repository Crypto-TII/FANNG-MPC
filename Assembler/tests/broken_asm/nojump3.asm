stms s0, 8201 # 4
jmp -3
stms s0, 8210 # 6
