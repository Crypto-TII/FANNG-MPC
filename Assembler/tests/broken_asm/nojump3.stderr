error: a jump jumped beyond the program's boundaries
 --> tests/broken_asm/nojump3.asm:2:1
  |
2 | jmp -3
  | ^^^^^^ jumped from here
  |